docs/getting_started.md

```diff
@@ -1,25 +1,33 @@
 # Getting Started
 
-Make sure you have pip and python 3.5 before starting
+Make sure you have pip and python 3.5 before continuing.
 
-## Benchmarks
- * Install Sanic
- * `python3 -m pip install sanic`
- * Edit main.py to include:
+## Installation
+
+ Install `sanic` by using the following command:
+
+ `python3 -m pip install sanic`
+
+It is as easy as that!
+
+## Hello World!
+
+ Create a file called `main.py` and paste the following code into.
 ```python
 from sanic import Sanic
-from sanic.response import json
+from sanic.response import text
 
 app = Sanic(__name__)
 
 @app.route("/")
 async def test(request):
-    return json({ "hello": "world" })
+    return text("Hello World!")
```

Comment:
I felt it would be much neater to view to a new user. Ideally its better if its html line like <h1>Hello World</h1> instead of both text or json response.